site/en/docs/webstore/using_webstore_api/index.md

```diff
@@ -2,36 +2,53 @@
-updating, and publishing items in the Chrome Web Store. Using this API, you can automate the process
-of uploading and publishing items into the store.
+updating, and publishing items in the Chrome Web Store.
 
 ## Before you begin {: #beforeyoubegin }
 
-To use the Chrome Web Store Publish API, you need to enable the API for your project in the [Google
-Developers Console][1].
-
-1.  Visit the [Google Developers Console][1].
-2.  Create a new project or select an existing one.
-3.  In the sidebar on the left, select **APIs & auth**.
-4.  In the displayed list of available APIs, set the status of the Chrome Web Store API to **ON**.
-5.  Accept the Terms of Service.
-6.  In the sidebar on the left, select **Credentials**.
-7.  Find the lines labeled **Client ID** and **Client secret**. Note that there may be a client ID
-    without a client secret for use with Compute Engine and App Engine. In that case, create a new
-    client ID and client secret.
-8.  To create the client ID and client secret, click on **CREATE CREDENTIALS**, select **OAuth client ID** and select **Desktop app** under **Application type**.
-9.  Get an access token:
-
-Once you have the client ID and client secret, you can retrieve an access token to work with the
-API. For example, enter this URL in your browser, replacing the \$CLIENT_ID with the one for your
-app:
+Take the following steps to get the access keys needed:
+
+**Enable the Chrome Web Store API**.
+
+1. Go to the [Google Cloud Console][google-dev-console].
+1. Create a new project or select an existing one.
+   {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/g5o7qpsDv5kKiQmlCwkv.png", alt="Create a new
+   project in the Google Console", width="600", height="149" %}  
+1. In the **search bar** type “Chrome Web Store API”.
+1. Enable the **Chrome Web Store API**.
+
+**Customize a consent screen**.
+
+1. Go to **OAuth consent screen**.
+1. Select **External** then **Create**.
+   {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/epcr782dUmXvZwaL5jid.png", alt="Create an Oauth
+   consent screen", width="547", height="573" %}
+1. Fill out the required **App information** fields (listed below) then click **Save and Continue**.
+    - App name.
+    - User Support email.
+    - Developer contact email.
+1. Skip Scopes. click **Save** then **Continue**.
+1. Add your email to **Test users**, then click **Save** then **Continue**.
+
+**Get the access keys**.
+
+1. Go to **Credentials**.
+1. Click **Create Credentials** then **OAuth client ID**.
+  {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/vZdPhBUnJytNg5W9TG0I.png", alt="Create credentials", width="657", height="106" %}  
+1. For **Application type**, choose **Desktop App**.
+1. Fill out the name, then click **Create**.
+
+The console will provide the client ID and client secret. You can retrieve an access token to work
+with the API. For example, enter this URL in your browser, replacing the $CLIENT_ID with the one for
+your app:
```

Comment:
I think it may be best to introduce another section titled "Testing your OAuth application" and to move everything between here and "Using the API" in that area. The last line of the previous section would be "The console will provide the client ID and client secret" and the first sentence of the new section would start with "You can retrieve…"

WDTY, @AmySteam?

Reply:
I like this suggestion! :)